src/View/TwigView.php

```diff
@@ -113,22 +113,17 @@ public function setTwig(Environment $twig): void
      */
     protected function resolveConfig(): array
     {
-        $charset = 'utf-8';
-        if (Configure::read('App.encoding') !== null) {
-            $charset = strtolower(Configure::read('App.encoding'));
-        }
-        $debugFlag = false;
-        if (Configure::read('App.encoding') === true) {
-            $debugFlag = true;
-        }
-        $config = [
-            'cache' => CACHE . 'twigView' . DS,
-            'charset' => $charset,
-            'auto_reload' => $debugFlag,
-            'debug' => $debugFlag,
+        $debug = Configure::read('debug', false);
+
+        $config = $this->readConfig() + [
+            'charset' => Configure::read('App.encoding', 'UTF-8'),
```

Comment:
```suggestion
            'charset' => Configure::read('App.encoding', 'utf-8'),
```

match the twig docs

Reply:
I always go to the source :)  https://github.com/twigphp/Twig/blob/6384ba6c3ca7695ef0aec93bdf3b55c9bb18b5de/src/Environment.php#L104